docs/api/components/schemas.yaml

```
@@ -1018,9 +1018,10 @@ Database:
               type: integer
               default: 60
             enable_star_channel:
-              description: Used to control whether Sync Gateway should use the all documents (*) channel.
+              description: Used to control whether Sync Gateway should use the all documents (*) channel. This is not configurable and the value is set to `true`.
```

```suggestion
              description: Since Sync Gateway 4.0,  this option no longer has an effect, the star channel is always enabled.
```

If the database will be marked as offline if this is true, then say:

```
description: Since Sync Gateway 4.0, this option no longer has an effect, the star channel is always enabled.

If this value is set to false, a database can only exist in the offline state until this configuration is modified.
```